frontend/src/models/analyses.js

```diff
@@ -357,15 +357,15 @@ const annotationRenderingTemporary = [
     'anchor': 'Casual_Variant',
     'header_datasets': [],
     'props': {
-      'attachmentDataset': 'Casual Variant In This Locus (ClinVar)',
-      'genomicAttachmentType': 'variant',
+      'attachmentDataset': 'Casual_Variant_In_This_Locus_ClinVar',
+      'genomicAttachmentType': 'hgvs_variant',
     },
     'rows': [{
       'class': '',
       'datasets': [
         {
-          'dataset': 'Casual Variant In This Locus (ClinVar)',
-          'genomicType': 'variant',
+          'dataset': 'Casual_Variant_In_This_Locus_ClinVar',
```

Comment:
```suggestion
          'dataset': 'Causal_Variant_In_This_Locus_ClinVar',
```
I completely spelled this wrong in the ticket.  It should be causal not casual.